### PR TITLE
[WIP]Bump CAPI version v1.1.3 into CAPA

### PR DESCRIPTION
**Pull request by shivi28**
**What type of PR is this?**
/area release

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

**Release note**:

```release-note
Bump CAPI version v1.1.3 into CAPA
```

### CODE REVIEW

File: test/e2e/data/e2e_eks_conf.yaml [hunk 1]
```diff
@@ -16,21 +16,20 @@ images:
   - name: gcr.io/k8s-staging-cluster-api/capa-manager:e2e
     loadBehavior: mustLoad
 
-## PLEASE KEEP THESE UP TO DATE WITH THE COMPONENTS
+  ## PLEASE KEEP THESE UP TO DATE WITH THE COMPONENTS
```
Comment (Ankitasw) [inline test/e2e/data/e2e_eks_conf.yaml:19]: nit: Indentation of new changes can be taken care (may be extra tabs are present)

Comment (shivi28) [reply test/e2e/data/e2e_eks_conf.yaml:19]: Yes I tried in both VSCode and Goland, its still coming

Comment (shivi28) [reply test/e2e/data/e2e_eks_conf.yaml:19]: yes checking

**Comment by shivi28** [comment]
/hold 
Will continue on this once after this [PR](https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/3253).

**Comment by pydctw** [comment]
Agree with @shivi28's assessment. Adding CAPA log as a reference
```
E0309 14:15:19.844182       1 awsmachinepool_controller.go:254]  "msg"="error updating AWSMachinePool" "error"="unable to update ASG: failed to update ASG \"machine-pool-liktoo-mp-0\": ValidationError: Desired capacity:0 must be between the specified min size:1 and max size:4\n\tstatus code: 400, request id: ab94e1af-159b-40d3-a95b-83cd6b78af0c" 
```

**Comment by sedefsavas** [comment]
/close

No longer needed.

**Pull request closed without merging**